bootstrap/starwhale/__init__.py

```diff
@@ -0,0 +1,61 @@
+import click
```

Comment:
bootstrap also use `starwhale` package name. if bootstrap and starwhale cli are installed in one python environemnt, they will influence each other ?